quranHome/src/commonMain/kotlin/com/islamversity/quran_home/model/JozUIModel.kt

```diff
@@ -0,0 +1,4 @@
+package com.islamversity.quran_home.model
+
+
+data class JozUIModel(val number: Int)
```

Comment:
if we keep passing long no more objects will be created (up to you)

Reply:
Sure, but I think if there is thinner type, why not?